04/components/Menu.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MenuItem from '../../03/components/MenuItem';
+
+const Menu = (props) => {
+  return (
+    <ul>
+      {props.items.map((item) => {
+        return <MenuItem text={item.text} url={item.url} />;
+      })}
```

Comment:
Można też wstawić to do zmiennej (`const items`) i tutaj osadzić tylko `<ul>{items}</ul>`.
To rozwiązanie wydaje mi się czytelniejsze ;)